jinahub/indexers/DocCache/doc_cache.py

```diff
@@ -87,7 +98,60 @@ def index_or_remove_from_request(self, docs: DocumentArray, **kwargs):
         for i in indices_to_remove:
             del docs[i]
         self.logger.info(
-            f"Finished index op. Cached doc ids: {len(self.cache_handler.id_to_hash)}; cached hashes: {len(self.cache_handler.hash_to_id)}"
+            f'Finished index op. Cached doc ids: {len(self.cache_handler.id_to_hash)}; cached hashes: {len(self.cache_handler.hash_to_id)}'
+        )
+
+    @requests(on='/update')
+    def update(self, docs: Optional[DocumentArray], **kwargs):
```

Comment:
Can you move them to their original position? Difficult to read the diff

Reply:
moved back